pytential/symbolic/execution.py

```diff
@@ -274,8 +274,9 @@ def map_common_subexpression(self, expr):
             return self.rec(expr.child)
 
         from numbers import Number
+        key = (expr.prefix, expr.child)
```

Comment:
How come the prefix became part of the key? (Add a comment to explain?)

Reply:
Ah, I'll remove that. It was initially added in the hopes of doing something more intelligent in the geometry merging (e.g. only copy over the expressions that have `DOFDescriptors` that are still in the collection)

Reply:
Added comment in https://github.com/inducer/pytential/compare/d623df0e16902c87f21c7bc05a7967db5a3e0836..5ba6baee5fa0bded87ab97ed9fc173d6b846aee2.